pandas/core/generic.py

```diff
@@ -1940,6 +1940,79 @@ def tail(self, n=5):
             return self
         return self.iloc[-n:]
 
+  
+    def sample(self, n=None, frac=None , replace=False, weights=None, seed=None):
+        """
+        Returns a sample of rows from object. 
```

Comment:
The word "random" should appear in here somewhere.

Perhaps `Sample rows at random from this object`?